Implementation for `test_clone_displays_nice_errors_for_missing_dependencies()` is broken on Windows

See use of `os.symlink()` below. Obviously, `tidy` does not live at `/usr/bin/tidy` on Windows! :roll_eyes: Finally! A bug that actually makes sense!

https://github.com/langfield/ki/blob/8a787f6ffb1dc21392a7d1e7ef70d4eda1eaaa3e/tests/test_integration.py#L481-L488

```python
The above exception was the direct cause of the following exception:

    def test_clone_displays_nice_errors_for_missing_dependencies():
        """Does it tell the user what to install?"""
        HTML: SampleCollection = get_test_collection("html")
        runner = CliRunner()
        with runner.isolated_filesystem():
    
            clone(runner, HTML.col_file)
            assert os.path.isdir(HTML.repodir)
            F.rmtree(F.test(Path(HTML.repodir)))
            old_path = os.environ["PATH"]
    
            # In case where nothing is installed, we expect to fail on `tidy`
            # first.
            try:
                with pytest.raises(FileNotFoundError) as raised:
                    os.environ["PATH"] = ""
                    clone(runner, HTML.col_file)
                error = raised.exconly()
                assert "tidy" in str(error)
            finally:
                os.environ["PATH"] = old_path
    
            # If `tidy` is on the PATH, but nothing else, then we expect a
            # `GitCommandNotFound` error.
            try:
                with pytest.raises(git.GitCommandNotFound) as raised:
                    tmp = F.mkdtemp()
                    os.symlink("/usr/bin/tidy", tmp / "tidy")
                    os.environ["PATH"] = str(tmp)
>                   clone(runner, HTML.col_file)

tests\test_integration.py:488: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
<@beartype(tests.test_ki.clone) at 0x214a42b1ab0>:85: in clone
    ???
tests\test_ki.py:310: in clone
    res = runner.invoke(
C:\hostedtoolcache\windows\Python\3.10.6\x64\lib\site-packages\click\testing.py:408: in invoke
    return_value = cli.main(args=args or (), prog_name=prog_name, **extra)
C:\hostedtoolcache\windows\Python\3.10.6\x64\lib\site-packages\click\core.py:1055: in main
    rv = self.invoke(ctx)
C:\hostedtoolcache\windows\Python\3.10.6\x64\lib\site-packages\click\core.py:1657: in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
C:\hostedtoolcache\windows\Python\3.10.6\x64\lib\site-packages\click\core.py:1404: in invoke
    return ctx.invoke(self.callback, **ctx.params)
C:\hostedtoolcache\windows\Python\3.10.6\x64\lib\site-packages\click\core.py:760: in invoke
    return __callback(*args, **kwargs)
ki\__init__.py:1747: in clone
    raise err
ki\__init__.py:1737: in clone
    _, _ = _clone(
<@beartype(ki._clone) at 0x214a3d995a0>:104: in _clone
    ???
ki\__init__.py:1806: in _clone
    write_repository(
<@beartype(ki.write_repository) at 0x214a3d98670>:124: in write_repository
    ???
ki\__init__.py:1093: in write_repository
    tidy_html_recursively(root, silent)
<@beartype(ki.tidy_html_recursively) at 0x214a3d98f70>:49: in tidy_html_recursively
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

root = EmptyDir('C:/Users/runneradmin/AppData/Local/Temp/tmpz8q0qexj/ki-fieldhtml')
silent = False

    @beartype
    def tidy_html_recursively(root: ExtantDir, silent: bool) -> None:
        """Call html5-tidy on each file in `root`, editing in-place."""
        # Spin up subprocesses for tidying field HTML in-place.
        batches: List[List[ExtantFile]] = list(
            F.get_batches(F.rglob(root, "*"), BATCH_SIZE)
        )
    
        bar = tqdm(batches, ncols=TQDM_NUM_COLS, leave=not silent)
        bar.set_description("HTML")
        for batch in bar:
            # TODO: Should we fail silently here, so as to not bother user with
            # tidy warnings?
            command = [
                "tidy",
                "-q",
                "-m",
                "-i",
                "-omit",
                "-utf8",
                "--tidy-mark",
                "no",
                "--show-body-only",
                "yes",
                "--wrap",
                "68",
                "--wrap-attributes",
                "yes",
            ]
            command += batch
            try:
                subprocess.run(command, check=False, capture_output=True)
            except FileNotFoundError as err:
>               raise MissingTidyExecutableError(err) from err
E               ki.types.MissingTidyExecutableError: Command not found: 'tidy' (Is 'html5-tidy' installed?)
E               Original exception: [WinError 2] The system cannot find the file
E               specified

ki\__init__.py:1515: MissingTidyExecutableError
```

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -92,6 +92,7 @@ EDITED: SampleCollection = get_test_collection("edited")
 # CLI
 
 
+@pytest.mark.skip
 def test_bad_command_is_bad():
     """Typos should result in errors."""
     result = invoke(ki.ki, ["clome"])
@@ -99,6 +100,7 @@ def test_bad_command_is_bad():
     assert "Error: No such command 'clome'." in result.output
 
 
+@pytest.mark.skip
 def test_runas_module():
     """Can this package be run as a Python module?"""
     command = "python -m ki --help"
@@ -106,12 +108,14 @@ def test_runas_module():
     assert completed.returncode == 0
 
 
+@pytest.mark.skip
 def test_entrypoint():
     """Is entrypoint script installed? (setup.py)"""
     result = invoke(ki.ki, ["--help"])
     assert result.exit_code == 0
 
 
+@pytest.mark.skip
 def test_version():
     """Does --version display information as expected?"""
     expected_version = version("ki")
@@ -121,6 +125,7 @@ def test_version():
     assert result.exit_code == 0
 
 
+@pytest.mark.skip
 def test_command_availability():
     """Are commands available?"""
     results = []
@@ -131,6 +136,7 @@ def test_command_availability():
         assert result.exit_code == 0
 
 
+@pytest.mark.skip
 def test_cli():
     """Does CLI stop execution w/o a command argument?"""
     with pytest.raises(SystemExit):
@@ -142,6 +148,7 @@ def test_cli():
 
 
 @beartype
+@pytest.mark.skip
 def test_fails_without_ki_subdirectory(tmp_path: Path):
     """Do pull and push know whether they're in a ki-generated git repo?"""
     runner = CliRunner()
@@ -156,6 +163,7 @@ def test_fails_without_ki_subdirectory(tmp_path: Path):
 
 
 @beartype
+@pytest.mark.skip
 def test_computes_and_stores_md5sum(tmp_path: Path):
     """Does ki add new hash to `.ki/hashes`?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -191,6 +199,7 @@ def test_computes_and_stores_md5sum(tmp_path: Path):
             assert f"199216c39eeabe23a1da016a99ffd3e2  {ORIGINAL.filename}" in hashes
 
 
+@pytest.mark.skip
 def test_no_op_pull_push_cycle_is_idempotent():
     """Do pull/push not misbehave if you keep doing both?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -216,6 +225,7 @@ def test_no_op_pull_push_cycle_is_idempotent():
         push(runner)
 
 
+@pytest.mark.skip
 def test_output(tmp_path: Path):
     """Does it print nice things?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -252,6 +262,7 @@ def test_output(tmp_path: Path):
 # CLONE
 
 
+@pytest.mark.skip
 def test_clone_fails_if_collection_doesnt_exist():
     """Does ki clone only if `.anki2` file exists?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -265,6 +276,7 @@ def test_clone_fails_if_collection_doesnt_exist():
         assert not os.path.isdir(ORIGINAL.repodir)
 
 
+@pytest.mark.skip
 def test_clone_fails_if_collection_is_already_open():
     """Does ki print a nice error message when Anki is accidentally left open?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -276,6 +288,7 @@ def test_clone_fails_if_collection_is_already_open():
             clone(runner, ORIGINAL.col_file)
 
 
+@pytest.mark.skip
 def test_clone_creates_directory():
     """Does it create the directory?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -289,6 +302,7 @@ def test_clone_creates_directory():
 
 
 @beartype
+@pytest.mark.skip
 def test_clone_displays_errors_from_creation_of_kirepo_metadata(mocker: MockerFixture):
     """Do errors get displayed nicely?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -304,6 +318,7 @@ def test_clone_displays_errors_from_creation_of_kirepo_metadata(mocker: MockerFi
 
 
 @beartype
+@pytest.mark.skip
 def test_clone_displays_errors_from_loading_kirepo_at_end(mocker: MockerFixture):
     """Do errors get propagated in the places we expect?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -354,6 +369,7 @@ def test_clone_displays_errors_from_loading_kirepo_at_end(mocker: MockerFixture)
             clone(runner, ORIGINAL.col_file)
 
 
+@pytest.mark.skip
 def test_clone_handles_html():
     """Does it tidy html and stuff?"""
     HTML: SampleCollection = get_test_collection("html")
@@ -372,6 +388,7 @@ def test_clone_handles_html():
         assert snippet in contents
 
 
+@pytest.mark.skip
 def test_clone_tidying_only_breaks_lines_for_fields_containing_html():
     """Does it tidy html and stuff?"""
     HTML: SampleCollection = get_test_collection("html")
@@ -392,6 +409,7 @@ def test_clone_tidying_only_breaks_lines_for_fields_containing_html():
         )
 
 
+@pytest.mark.skip
 def test_clone_errors_when_directory_is_populated():
     """Does it disallow overwrites?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -410,6 +428,7 @@ def test_clone_errors_when_directory_is_populated():
             clone(runner, ORIGINAL.col_file)
 
 
+@pytest.mark.skip
 def test_clone_cleans_up_on_error():
     """Does it clean up on nontrivial errors?"""
     HTML: SampleCollection = get_test_collection("html")
@@ -429,6 +448,7 @@ def test_clone_cleans_up_on_error():
             os.environ["PATH"] = old_path
 
 
+@pytest.mark.skip
 def test_clone_cleans_up_preserves_directories_that_exist_a_priori():
     """Does clone not delete targetdirs that already existed?"""
     HTML: SampleCollection = get_test_collection("html")
@@ -485,6 +505,7 @@ def test_clone_displays_nice_errors_for_missing_dependencies():
             os.environ["PATH"] = old_path
 
 
+@pytest.mark.skip
 def test_clone_succeeds_when_directory_exists_but_is_empty():
     """Does it clone into empty directories?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -496,6 +517,7 @@ def test_clone_succeeds_when_directory_exists_but_is_empty():
         clone(runner, ORIGINAL.col_file)
 
 
+@pytest.mark.skip
 def test_clone_generates_expected_notes():
     """Do generated note files match content of an example collection?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -517,6 +539,7 @@ def test_clone_generates_expected_notes():
         assert cloned_md5 == true_md5
 
 
+@pytest.mark.skip
 def test_clone_generates_deck_tree_correctly():
     """Does generated FS tree match example collection?"""
     MULTIDECK: SampleCollection = get_test_collection("multideck")
@@ -540,6 +563,7 @@ def test_clone_generates_deck_tree_correctly():
         assert cloned_md5 == true_md5
 
 
+@pytest.mark.skip
 def test_clone_generates_ki_subdirectory():
     """Does clone command generate .ki/ directory?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -554,6 +578,7 @@ def test_clone_generates_ki_subdirectory():
         assert os.path.isdir(kidir)
 
 
+@pytest.mark.skip
 def test_cloned_collection_is_git_repository():
     """Does clone run `git init` and stuff?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -566,6 +591,7 @@ def test_cloned_collection_is_git_repository():
         assert is_git_repo(ORIGINAL.repodir)
 
 
+@pytest.mark.skip
 def test_clone_commits_directory_contents():
     """Does clone leave user with an up-to-date repo?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -587,6 +613,7 @@ def test_clone_commits_directory_contents():
         assert len(commits) == 1
 
 
+@pytest.mark.skip
 def test_clone_leaves_collection_file_unchanged():
     """Does clone leave the collection alone?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -601,6 +628,7 @@ def test_clone_leaves_collection_file_unchanged():
         assert original_md5 == updated_md5
 
 
+@pytest.mark.skip
 def test_clone_directory_argument_works():
     """Does clone obey the target directory argument?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -617,6 +645,7 @@ def test_clone_directory_argument_works():
         assert os.path.isdir(target)
 
 
+@pytest.mark.skip
 def test_clone_writes_media_files():
     """Does clone copy media files from the media directory into 'MEDIA'?"""
     MEDIACOL: SampleCollection = get_test_collection("media")
@@ -629,6 +658,7 @@ def test_clone_writes_media_files():
         assert audio_path.is_file()
 
 
+@pytest.mark.skip
 def test_clone_handles_cards_from_a_single_note_in_distinct_decks(tmp_path: Path):
     SPLIT: SampleCollection = get_test_collection("split")
     runner = CliRunner()
@@ -638,6 +668,7 @@ def test_clone_handles_cards_from_a_single_note_in_distinct_decks(tmp_path: Path
         assert os.path.isfile(Path(SPLIT.repodir) / "top" / "a" / "a.md")
 
 
+@pytest.mark.skip
 def test_clone_url_decodes_media_src_attributes(tmp_path: Path):
     DOUBLE: SampleCollection = get_test_collection("no_double_encodings")
     runner = CliRunner()
@@ -654,6 +685,7 @@ def test_clone_url_decodes_media_src_attributes(tmp_path: Path):
 # PULL
 
 
+@pytest.mark.skip
 def test_pull_fails_if_collection_no_longer_exists():
     """Does ki pull only if `.anki2` file exists?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -670,6 +702,7 @@ def test_pull_fails_if_collection_no_longer_exists():
             pull(runner)
 
 
+@pytest.mark.skip
 def test_pull_fails_if_collection_file_is_corrupted():
     """Does `pull()` fail gracefully when the collection file is bad?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -687,6 +720,7 @@ def test_pull_fails_if_collection_file_is_corrupted():
             pull(runner)
 
 
+@pytest.mark.skip
 def test_pull_writes_changes_correctly():
     """Does ki get the changes from modified collection file?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -706,6 +740,7 @@ def test_pull_writes_changes_correctly():
         assert os.path.isfile(NOTE_1)
 
 
+@pytest.mark.skip
 def test_pull_unchanged_collection_is_no_op():
     """Does ki remove remote before quitting?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -725,6 +760,7 @@ def test_pull_unchanged_collection_is_no_op():
         assert orig_hash == new_hash
 
 
+@pytest.mark.skip
 def test_pull_avoids_unnecessary_merge_conflicts():
     """Does ki prevent gratuitous merge conflicts?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -744,6 +780,7 @@ def test_pull_avoids_unnecessary_merge_conflicts():
         assert "Automatic merge failed; fix" not in out
 
 
+@pytest.mark.skip
 def test_pull_still_works_from_subdirectories():
     """Does pull still work if you're farther down in the directory tree than the repo route?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -762,6 +799,7 @@ def test_pull_still_works_from_subdirectories():
         pull(runner)
 
 
+@pytest.mark.skip
 def test_pull_displays_errors_from_repo_ref():
     """Does 'pull()' return early when the last push commit ref is bad?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -782,6 +820,7 @@ def test_pull_displays_errors_from_repo_ref():
             pull(runner)
 
 
+@pytest.mark.skip
 def test_pull_displays_errors_from_clone_helper(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -802,6 +841,7 @@ def test_pull_displays_errors_from_clone_helper(mocker: MockerFixture):
             pull(runner)
 
 
+@pytest.mark.skip
 def test_pull_handles_unexpectedly_changed_checksums(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -820,6 +860,7 @@ def test_pull_handles_unexpectedly_changed_checksums(mocker: MockerFixture):
             pull(runner)
 
 
+@pytest.mark.skip
 def test_pull_displays_errors_from_repo_initialization(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -841,6 +882,7 @@ def test_pull_displays_errors_from_repo_initialization(mocker: MockerFixture):
 
 
 @pytest.mark.xfail
+@pytest.mark.skip
 def test_pull_preserves_reassigned_note_ids(tmp_path: Path):
     """UNFINISHED!"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -871,6 +913,7 @@ def test_pull_preserves_reassigned_note_ids(tmp_path: Path):
         raise NotImplementedError
 
 
+@pytest.mark.skip
 def test_pull_handles_non_standard_submodule_branch_names(tmp_path: Path):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -900,6 +943,7 @@ def test_pull_handles_non_standard_submodule_branch_names(tmp_path: Path):
         out = pull(runner)
 
 
+@pytest.mark.skip
 def test_pull_handles_uncommitted_submodule_commits(tmp_path: Path):
     UNCOMMITTED_SM: SampleCollection = get_test_collection(
         "uncommitted_submodule_commits"
@@ -983,6 +1027,7 @@ def test_pull_handles_uncommitted_submodule_commits(tmp_path: Path):
         assert expected_this in note_text
 
 
+@pytest.mark.skip
 def test_pull_removes_files_deleted_in_remote(tmp_path: Path):
     ORIGINAL: SampleCollection = get_test_collection("original")
     DELETED: SampleCollection = get_test_collection("deleted")
@@ -999,6 +1044,7 @@ def test_pull_removes_files_deleted_in_remote(tmp_path: Path):
         out = pull(runner)
 
 
+@pytest.mark.skip
 def test_pull_does_not_duplicate_decks_converted_to_subdecks_of_new_top_level_decks(
     tmp_path: Path,
 ):
@@ -1026,6 +1072,7 @@ def test_pull_does_not_duplicate_decks_converted_to_subdecks_of_new_top_level_de
 # PUSH
 
 
+@pytest.mark.skip
 def test_push_writes_changes_correctly(tmp_path: Path):
     """If there are committed changes, does push change the collection file?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1080,6 +1127,7 @@ def test_push_writes_changes_correctly(tmp_path: Path):
         assert len(new_notes) == 2
 
 
+@pytest.mark.skip
 def test_push_verifies_md5sum():
     """Does ki only push if md5sum matches last pull?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1098,6 +1146,7 @@ def test_push_verifies_md5sum():
             push(runner)
 
 
+@pytest.mark.skip
 def test_push_generates_correct_backup():
     """Does push store a backup identical to old collection file?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1133,6 +1182,7 @@ def test_push_generates_correct_backup():
         assert backup_exists
 
 
+@pytest.mark.skip
 def test_push_doesnt_write_uncommitted_changes():
     """Does push only write changes that have been committed?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1154,6 +1204,7 @@ def test_push_doesnt_write_uncommitted_changes():
         assert len(os.listdir(".ki/backups")) == 0
 
 
+@pytest.mark.skip
 def test_push_doesnt_fail_after_pull():
     """Does push work if we pull and then edit and then push?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1193,6 +1244,7 @@ def test_push_doesnt_fail_after_pull():
         push(runner)
 
 
+@pytest.mark.skip
 def test_no_op_push_is_idempotent():
     """Does push not misbehave if you keep pushing?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1212,6 +1264,7 @@ def test_no_op_push_is_idempotent():
         push(runner)
 
 
+@pytest.mark.skip
 def test_push_deletes_notes():
     """Does push remove deleted notes from collection?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1242,6 +1295,7 @@ def test_push_deletes_notes():
         assert not os.path.isfile(NOTE_0)
 
 
+@pytest.mark.skip
 def test_push_still_works_from_subdirectories():
     """Does push still work if you're farther down in the directory tree than the repo route?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1267,6 +1321,7 @@ def test_push_still_works_from_subdirectories():
         push(runner)
 
 
+@pytest.mark.skip
 def test_push_deletes_added_notes():
     """Does push remove deleted notes added with ki?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1324,6 +1379,7 @@ def test_push_deletes_added_notes():
 # We expect this test not to work anymore, because it is now outside the scope
 # of `ki`.
 @pytest.mark.xfail
+@pytest.mark.skip
 def test_push_generates_correct_title_for_notes():
     """Does push use the truncated sort field as a filename?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1355,6 +1411,7 @@ def test_push_generates_correct_title_for_notes():
         assert "r.md" in notes
 
 
+@pytest.mark.skip
 def test_push_displays_informative_error_when_last_push_file_is_missing():
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1373,6 +1430,7 @@ def test_push_displays_informative_error_when_last_push_file_is_missing():
             push(runner)
 
 
+@pytest.mark.skip
 def test_push_honors_ignore_patterns():
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1408,6 +1466,7 @@ def test_push_honors_ignore_patterns():
         assert "Warning: not Anki note" in out
 
 
+@pytest.mark.skip
 def test_push_displays_errors_from_head_ref_maybes(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1429,6 +1488,7 @@ def test_push_displays_errors_from_head_ref_maybes(mocker: MockerFixture):
             push(runner)
 
 
+@pytest.mark.skip
 def test_push_displays_errors_from_head_repo_ref(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1452,6 +1512,7 @@ def test_push_displays_errors_from_head_repo_ref(mocker: MockerFixture):
             push(runner)
 
 
+@pytest.mark.skip
 def test_push_displays_errors_from_notetype_parsing_in_push_deltas_during_model_adding(
     mocker: MockerFixture,
 ):
@@ -1483,6 +1544,7 @@ def test_push_displays_errors_from_notetype_parsing_in_push_deltas_during_model_
             push(runner)
 
 
+@pytest.mark.skip
 def test_push_displays_errors_from_notetype_parsing_in_push_deltas_during_push_flatnote_to_anki(
     mocker: MockerFixture,
 ):
@@ -1515,6 +1577,7 @@ def test_push_displays_errors_from_notetype_parsing_in_push_deltas_during_push_f
             out = push(runner)
 
 
+@pytest.mark.skip
 def test_push_handles_submodules(tmp_path: Path):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1546,6 +1609,7 @@ def test_push_handles_submodules(tmp_path: Path):
         assert "<br>z<br>" in notes[0]["Back"]
 
 
+@pytest.mark.skip
 def test_push_writes_media(tmp_path: Path):
     MEDIACOL: SampleCollection = get_test_collection("media")
     runner = CliRunner()
@@ -1587,6 +1651,7 @@ def test_push_writes_media(tmp_path: Path):
         assert len(check.unused) == 0
 
 
+@pytest.mark.skip
 def test_push_handles_foreign_models(tmp_path: Path):
     """Just check that we don't return an exception from `push()`."""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1602,6 +1667,7 @@ def test_push_handles_foreign_models(tmp_path: Path):
         out = push(runner)
 
 
+@pytest.mark.skip
 def test_push_fails_if_database_is_locked():
     """Does ki print a nice error message when Anki is accidentally left open?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1621,6 +1687,7 @@ def test_push_fails_if_database_is_locked():
             push(runner)
 
 
+@pytest.mark.skip
 def test_push_is_nontrivial_when_pulled_changes_are_reverted(tmp_path: Path):
     """
     If you push, make changes in Anki, then pull those changes, then undo them
@@ -1676,6 +1743,7 @@ def test_push_is_nontrivial_when_pulled_changes_are_reverted(tmp_path: Path):
         assert "ki push: up to date." not in out
 
 
+@pytest.mark.skip
 def test_push_doesnt_unnecessarily_deduplicate_notetypes():
     """
     Does push refrain from adding a new notetype if the requested notetype
@@ -1731,6 +1799,7 @@ def test_push_doesnt_unnecessarily_deduplicate_notetypes():
         col.close(save=False)
 
 
+@pytest.mark.skip
 def test_push_is_nontrivial_when_pushed_changes_are_reverted_in_repository():
     """
     The following operation should be nontrivial:
@@ -1776,6 +1845,7 @@ def test_push_is_nontrivial_when_pushed_changes_are_reverted_in_repository():
         assert "ki push: up to date." not in out
 
 
+@pytest.mark.skip
 def test_push_changes_deck_for_moved_notes():
     MULTIDECK: SampleCollection = get_test_collection("multideck")
     runner = CliRunner()
@@ -1810,6 +1880,7 @@ def test_push_changes_deck_for_moved_notes():
         assert colnote.deck == "aa::dd"
 
 
+@pytest.mark.skip
 def test_push_is_trivial_for_committed_submodule_contents(tmp_path: Path):
     UNCOMMITTED_SM: SampleCollection = get_test_collection(
         "uncommitted_submodule_commits"
@@ -1850,6 +1921,7 @@ def test_push_is_trivial_for_committed_submodule_contents(tmp_path: Path):
         assert "ki push: up to date." in out
 
 
+@pytest.mark.skip
 def test_push_prints_informative_warning_on_push_when_subrepo_was_added_instead_of_submodule(
     tmp_path: Path,
 ):
@@ -1879,6 +1951,7 @@ def test_push_prints_informative_warning_on_push_when_subrepo_was_added_instead_
             assert "'git submodule add'" in out
 
 
+@pytest.mark.skip
 def test_push_handles_tags_containing_trailing_commas():
     COMMAS: SampleCollection = get_test_collection("commas")
     runner = CliRunner()
@@ -1902,6 +1975,7 @@ def test_push_handles_tags_containing_trailing_commas():
         push(runner)
 
 
+@pytest.mark.skip
 def test_push_correctly_encodes_quotes_in_html_tags(tmp_path: Path):
     BROKEN: SampleCollection = get_test_collection("broken_media_links")
     runner = CliRunner()

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 """Tests for ki command line interface (CLI)."""
 import os
+import sys
 import shutil
 import sqlite3
 import tempfile

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -93,7 +93,6 @@ EDITED: SampleCollection = get_test_collection("edited")
 # CLI
 
 
-@pytest.mark.skip
 def test_bad_command_is_bad():
     """Typos should result in errors."""
     result = invoke(ki.ki, ["clome"])
@@ -101,7 +100,6 @@ def test_bad_command_is_bad():
     assert "Error: No such command 'clome'." in result.output
 
 
-@pytest.mark.skip
 def test_runas_module():
     """Can this package be run as a Python module?"""
     command = "python -m ki --help"
@@ -109,14 +107,12 @@ def test_runas_module():
     assert completed.returncode == 0
 
 
-@pytest.mark.skip
 def test_entrypoint():
     """Is entrypoint script installed? (setup.py)"""
     result = invoke(ki.ki, ["--help"])
     assert result.exit_code == 0
 
 
-@pytest.mark.skip
 def test_version():
     """Does --version display information as expected?"""
     expected_version = version("ki")
@@ -126,7 +122,6 @@ def test_version():
     assert result.exit_code == 0
 
 
-@pytest.mark.skip
 def test_command_availability():
     """Are commands available?"""
     results = []
@@ -137,7 +132,6 @@ def test_command_availability():
         assert result.exit_code == 0
 
 
-@pytest.mark.skip
 def test_cli():
     """Does CLI stop execution w/o a command argument?"""
     with pytest.raises(SystemExit):
@@ -149,7 +143,6 @@ def test_cli():
 
 
 @beartype
-@pytest.mark.skip
 def test_fails_without_ki_subdirectory(tmp_path: Path):
     """Do pull and push know whether they're in a ki-generated git repo?"""
     runner = CliRunner()
@@ -164,7 +157,6 @@ def test_fails_without_ki_subdirectory(tmp_path: Path):
 
 
 @beartype
-@pytest.mark.skip
 def test_computes_and_stores_md5sum(tmp_path: Path):
     """Does ki add new hash to `.ki/hashes`?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -200,7 +192,6 @@ def test_computes_and_stores_md5sum(tmp_path: Path):
             assert f"199216c39eeabe23a1da016a99ffd3e2  {ORIGINAL.filename}" in hashes
 
 
-@pytest.mark.skip
 def test_no_op_pull_push_cycle_is_idempotent():
     """Do pull/push not misbehave if you keep doing both?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -226,7 +217,6 @@ def test_no_op_pull_push_cycle_is_idempotent():
         push(runner)
 
 
-@pytest.mark.skip
 def test_output(tmp_path: Path):
     """Does it print nice things?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -263,7 +253,6 @@ def test_output(tmp_path: Path):
 # CLONE
 
 
-@pytest.mark.skip
 def test_clone_fails_if_collection_doesnt_exist():
     """Does ki clone only if `.anki2` file exists?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -277,7 +266,6 @@ def test_clone_fails_if_collection_doesnt_exist():
         assert not os.path.isdir(ORIGINAL.repodir)
 
 
-@pytest.mark.skip
 def test_clone_fails_if_collection_is_already_open():
     """Does ki print a nice error message when Anki is accidentally left open?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -289,7 +277,6 @@ def test_clone_fails_if_collection_is_already_open():
             clone(runner, ORIGINAL.col_file)
 
 
-@pytest.mark.skip
 def test_clone_creates_directory():
     """Does it create the directory?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -303,7 +290,6 @@ def test_clone_creates_directory():
 
 
 @beartype
-@pytest.mark.skip
 def test_clone_displays_errors_from_creation_of_kirepo_metadata(mocker: MockerFixture):
     """Do errors get displayed nicely?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -319,7 +305,6 @@ def test_clone_displays_errors_from_creation_of_kirepo_metadata(mocker: MockerFi
 
 
 @beartype
-@pytest.mark.skip
 def test_clone_displays_errors_from_loading_kirepo_at_end(mocker: MockerFixture):
     """Do errors get propagated in the places we expect?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -370,7 +355,6 @@ def test_clone_displays_errors_from_loading_kirepo_at_end(mocker: MockerFixture)
             clone(runner, ORIGINAL.col_file)
 
 
-@pytest.mark.skip
 def test_clone_handles_html():
     """Does it tidy html and stuff?"""
     HTML: SampleCollection = get_test_collection("html")
@@ -389,7 +373,6 @@ def test_clone_handles_html():
         assert snippet in contents
 
 
-@pytest.mark.skip
 def test_clone_tidying_only_breaks_lines_for_fields_containing_html():
     """Does it tidy html and stuff?"""
     HTML: SampleCollection = get_test_collection("html")
@@ -410,7 +393,6 @@ def test_clone_tidying_only_breaks_lines_for_fields_containing_html():
         )
 
 
-@pytest.mark.skip
 def test_clone_errors_when_directory_is_populated():
     """Does it disallow overwrites?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -429,7 +411,6 @@ def test_clone_errors_when_directory_is_populated():
             clone(runner, ORIGINAL.col_file)
 
 
-@pytest.mark.skip
 def test_clone_cleans_up_on_error():
     """Does it clean up on nontrivial errors?"""
     HTML: SampleCollection = get_test_collection("html")
@@ -449,7 +430,6 @@ def test_clone_cleans_up_on_error():
             os.environ["PATH"] = old_path
 
 
-@pytest.mark.skip
 def test_clone_cleans_up_preserves_directories_that_exist_a_priori():
     """Does clone not delete targetdirs that already existed?"""
     HTML: SampleCollection = get_test_collection("html")
@@ -510,7 +490,6 @@ def test_clone_displays_nice_errors_for_missing_dependencies():
             os.environ["PATH"] = old_path
 
 
-@pytest.mark.skip
 def test_clone_succeeds_when_directory_exists_but_is_empty():
     """Does it clone into empty directories?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -522,7 +501,6 @@ def test_clone_succeeds_when_directory_exists_but_is_empty():
         clone(runner, ORIGINAL.col_file)
 
 
-@pytest.mark.skip
 def test_clone_generates_expected_notes():
     """Do generated note files match content of an example collection?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -544,7 +522,6 @@ def test_clone_generates_expected_notes():
         assert cloned_md5 == true_md5
 
 
-@pytest.mark.skip
 def test_clone_generates_deck_tree_correctly():
     """Does generated FS tree match example collection?"""
     MULTIDECK: SampleCollection = get_test_collection("multideck")
@@ -568,7 +545,6 @@ def test_clone_generates_deck_tree_correctly():
         assert cloned_md5 == true_md5
 
 
-@pytest.mark.skip
 def test_clone_generates_ki_subdirectory():
     """Does clone command generate .ki/ directory?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -583,7 +559,6 @@ def test_clone_generates_ki_subdirectory():
         assert os.path.isdir(kidir)
 
 
-@pytest.mark.skip
 def test_cloned_collection_is_git_repository():
     """Does clone run `git init` and stuff?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -596,7 +571,6 @@ def test_cloned_collection_is_git_repository():
         assert is_git_repo(ORIGINAL.repodir)
 
 
-@pytest.mark.skip
 def test_clone_commits_directory_contents():
     """Does clone leave user with an up-to-date repo?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -618,7 +592,6 @@ def test_clone_commits_directory_contents():
         assert len(commits) == 1
 
 
-@pytest.mark.skip
 def test_clone_leaves_collection_file_unchanged():
     """Does clone leave the collection alone?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -633,7 +606,6 @@ def test_clone_leaves_collection_file_unchanged():
         assert original_md5 == updated_md5
 
 
-@pytest.mark.skip
 def test_clone_directory_argument_works():
     """Does clone obey the target directory argument?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -650,7 +622,6 @@ def test_clone_directory_argument_works():
         assert os.path.isdir(target)
 
 
-@pytest.mark.skip
 def test_clone_writes_media_files():
     """Does clone copy media files from the media directory into 'MEDIA'?"""
     MEDIACOL: SampleCollection = get_test_collection("media")
@@ -663,7 +634,6 @@ def test_clone_writes_media_files():
         assert audio_path.is_file()
 
 
-@pytest.mark.skip
 def test_clone_handles_cards_from_a_single_note_in_distinct_decks(tmp_path: Path):
     SPLIT: SampleCollection = get_test_collection("split")
     runner = CliRunner()
@@ -673,7 +643,6 @@ def test_clone_handles_cards_from_a_single_note_in_distinct_decks(tmp_path: Path
         assert os.path.isfile(Path(SPLIT.repodir) / "top" / "a" / "a.md")
 
 
-@pytest.mark.skip
 def test_clone_url_decodes_media_src_attributes(tmp_path: Path):
     DOUBLE: SampleCollection = get_test_collection("no_double_encodings")
     runner = CliRunner()
@@ -690,7 +659,6 @@ def test_clone_url_decodes_media_src_attributes(tmp_path: Path):
 # PULL
 
 
-@pytest.mark.skip
 def test_pull_fails_if_collection_no_longer_exists():
     """Does ki pull only if `.anki2` file exists?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -707,7 +675,6 @@ def test_pull_fails_if_collection_no_longer_exists():
             pull(runner)
 
 
-@pytest.mark.skip
 def test_pull_fails_if_collection_file_is_corrupted():
     """Does `pull()` fail gracefully when the collection file is bad?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -725,7 +692,6 @@ def test_pull_fails_if_collection_file_is_corrupted():
             pull(runner)
 
 
-@pytest.mark.skip
 def test_pull_writes_changes_correctly():
     """Does ki get the changes from modified collection file?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -745,7 +711,6 @@ def test_pull_writes_changes_correctly():
         assert os.path.isfile(NOTE_1)
 
 
-@pytest.mark.skip
 def test_pull_unchanged_collection_is_no_op():
     """Does ki remove remote before quitting?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -765,7 +730,6 @@ def test_pull_unchanged_collection_is_no_op():
         assert orig_hash == new_hash
 
 
-@pytest.mark.skip
 def test_pull_avoids_unnecessary_merge_conflicts():
     """Does ki prevent gratuitous merge conflicts?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -785,7 +749,6 @@ def test_pull_avoids_unnecessary_merge_conflicts():
         assert "Automatic merge failed; fix" not in out
 
 
-@pytest.mark.skip
 def test_pull_still_works_from_subdirectories():
     """Does pull still work if you're farther down in the directory tree than the repo route?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -804,7 +767,6 @@ def test_pull_still_works_from_subdirectories():
         pull(runner)
 
 
-@pytest.mark.skip
 def test_pull_displays_errors_from_repo_ref():
     """Does 'pull()' return early when the last push commit ref is bad?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -825,7 +787,6 @@ def test_pull_displays_errors_from_repo_ref():
             pull(runner)
 
 
-@pytest.mark.skip
 def test_pull_displays_errors_from_clone_helper(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -846,7 +807,6 @@ def test_pull_displays_errors_from_clone_helper(mocker: MockerFixture):
             pull(runner)
 
 
-@pytest.mark.skip
 def test_pull_handles_unexpectedly_changed_checksums(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -865,7 +825,6 @@ def test_pull_handles_unexpectedly_changed_checksums(mocker: MockerFixture):
             pull(runner)
 
 
-@pytest.mark.skip
 def test_pull_displays_errors_from_repo_initialization(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -887,7 +846,6 @@ def test_pull_displays_errors_from_repo_initialization(mocker: MockerFixture):
 
 
 @pytest.mark.xfail
-@pytest.mark.skip
 def test_pull_preserves_reassigned_note_ids(tmp_path: Path):
     """UNFINISHED!"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -918,7 +876,6 @@ def test_pull_preserves_reassigned_note_ids(tmp_path: Path):
         raise NotImplementedError
 
 
-@pytest.mark.skip
 def test_pull_handles_non_standard_submodule_branch_names(tmp_path: Path):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -948,7 +905,6 @@ def test_pull_handles_non_standard_submodule_branch_names(tmp_path: Path):
         out = pull(runner)
 
 
-@pytest.mark.skip
 def test_pull_handles_uncommitted_submodule_commits(tmp_path: Path):
     UNCOMMITTED_SM: SampleCollection = get_test_collection(
         "uncommitted_submodule_commits"
@@ -1032,7 +988,6 @@ def test_pull_handles_uncommitted_submodule_commits(tmp_path: Path):
         assert expected_this in note_text
 
 
-@pytest.mark.skip
 def test_pull_removes_files_deleted_in_remote(tmp_path: Path):
     ORIGINAL: SampleCollection = get_test_collection("original")
     DELETED: SampleCollection = get_test_collection("deleted")
@@ -1049,7 +1004,6 @@ def test_pull_removes_files_deleted_in_remote(tmp_path: Path):
         out = pull(runner)
 
 
-@pytest.mark.skip
 def test_pull_does_not_duplicate_decks_converted_to_subdecks_of_new_top_level_decks(
     tmp_path: Path,
 ):
@@ -1077,7 +1031,6 @@ def test_pull_does_not_duplicate_decks_converted_to_subdecks_of_new_top_level_de
 # PUSH
 
 
-@pytest.mark.skip
 def test_push_writes_changes_correctly(tmp_path: Path):
     """If there are committed changes, does push change the collection file?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1132,7 +1085,6 @@ def test_push_writes_changes_correctly(tmp_path: Path):
         assert len(new_notes) == 2
 
 
-@pytest.mark.skip
 def test_push_verifies_md5sum():
     """Does ki only push if md5sum matches last pull?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1151,7 +1103,6 @@ def test_push_verifies_md5sum():
             push(runner)
 
 
-@pytest.mark.skip
 def test_push_generates_correct_backup():
     """Does push store a backup identical to old collection file?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1187,7 +1138,6 @@ def test_push_generates_correct_backup():
         assert backup_exists
 
 
-@pytest.mark.skip
 def test_push_doesnt_write_uncommitted_changes():
     """Does push only write changes that have been committed?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1209,7 +1159,6 @@ def test_push_doesnt_write_uncommitted_changes():
         assert len(os.listdir(".ki/backups")) == 0
 
 
-@pytest.mark.skip
 def test_push_doesnt_fail_after_pull():
     """Does push work if we pull and then edit and then push?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1249,7 +1198,6 @@ def test_push_doesnt_fail_after_pull():
         push(runner)
 
 
-@pytest.mark.skip
 def test_no_op_push_is_idempotent():
     """Does push not misbehave if you keep pushing?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1269,7 +1217,6 @@ def test_no_op_push_is_idempotent():
         push(runner)
 
 
-@pytest.mark.skip
 def test_push_deletes_notes():
     """Does push remove deleted notes from collection?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1300,7 +1247,6 @@ def test_push_deletes_notes():
         assert not os.path.isfile(NOTE_0)
 
 
-@pytest.mark.skip
 def test_push_still_works_from_subdirectories():
     """Does push still work if you're farther down in the directory tree than the repo route?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1326,7 +1272,6 @@ def test_push_still_works_from_subdirectories():
         push(runner)
 
 
-@pytest.mark.skip
 def test_push_deletes_added_notes():
     """Does push remove deleted notes added with ki?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1384,7 +1329,6 @@ def test_push_deletes_added_notes():
 # We expect this test not to work anymore, because it is now outside the scope
 # of `ki`.
 @pytest.mark.xfail
-@pytest.mark.skip
 def test_push_generates_correct_title_for_notes():
     """Does push use the truncated sort field as a filename?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1416,7 +1360,6 @@ def test_push_generates_correct_title_for_notes():
         assert "r.md" in notes
 
 
-@pytest.mark.skip
 def test_push_displays_informative_error_when_last_push_file_is_missing():
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1435,7 +1378,6 @@ def test_push_displays_informative_error_when_last_push_file_is_missing():
             push(runner)
 
 
-@pytest.mark.skip
 def test_push_honors_ignore_patterns():
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1471,7 +1413,6 @@ def test_push_honors_ignore_patterns():
         assert "Warning: not Anki note" in out
 
 
-@pytest.mark.skip
 def test_push_displays_errors_from_head_ref_maybes(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1493,7 +1434,6 @@ def test_push_displays_errors_from_head_ref_maybes(mocker: MockerFixture):
             push(runner)
 
 
-@pytest.mark.skip
 def test_push_displays_errors_from_head_repo_ref(mocker: MockerFixture):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1517,7 +1457,6 @@ def test_push_displays_errors_from_head_repo_ref(mocker: MockerFixture):
             push(runner)
 
 
-@pytest.mark.skip
 def test_push_displays_errors_from_notetype_parsing_in_push_deltas_during_model_adding(
     mocker: MockerFixture,
 ):
@@ -1549,7 +1488,6 @@ def test_push_displays_errors_from_notetype_parsing_in_push_deltas_during_model_
             push(runner)
 
 
-@pytest.mark.skip
 def test_push_displays_errors_from_notetype_parsing_in_push_deltas_during_push_flatnote_to_anki(
     mocker: MockerFixture,
 ):
@@ -1582,7 +1520,6 @@ def test_push_displays_errors_from_notetype_parsing_in_push_deltas_during_push_f
             out = push(runner)
 
 
-@pytest.mark.skip
 def test_push_handles_submodules(tmp_path: Path):
     ORIGINAL: SampleCollection = get_test_collection("original")
     runner = CliRunner()
@@ -1614,7 +1551,6 @@ def test_push_handles_submodules(tmp_path: Path):
         assert "<br>z<br>" in notes[0]["Back"]
 
 
-@pytest.mark.skip
 def test_push_writes_media(tmp_path: Path):
     MEDIACOL: SampleCollection = get_test_collection("media")
     runner = CliRunner()
@@ -1656,7 +1592,6 @@ def test_push_writes_media(tmp_path: Path):
         assert len(check.unused) == 0
 
 
-@pytest.mark.skip
 def test_push_handles_foreign_models(tmp_path: Path):
     """Just check that we don't return an exception from `push()`."""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1672,7 +1607,6 @@ def test_push_handles_foreign_models(tmp_path: Path):
         out = push(runner)
 
 
-@pytest.mark.skip
 def test_push_fails_if_database_is_locked():
     """Does ki print a nice error message when Anki is accidentally left open?"""
     ORIGINAL: SampleCollection = get_test_collection("original")
@@ -1692,7 +1626,6 @@ def test_push_fails_if_database_is_locked():
             push(runner)
 
 
-@pytest.mark.skip
 def test_push_is_nontrivial_when_pulled_changes_are_reverted(tmp_path: Path):
     """
     If you push, make changes in Anki, then pull those changes, then undo them
@@ -1748,7 +1681,6 @@ def test_push_is_nontrivial_when_pulled_changes_are_reverted(tmp_path: Path):
         assert "ki push: up to date." not in out
 
 
-@pytest.mark.skip
 def test_push_doesnt_unnecessarily_deduplicate_notetypes():
     """
     Does push refrain from adding a new notetype if the requested notetype
@@ -1804,7 +1736,6 @@ def test_push_doesnt_unnecessarily_deduplicate_notetypes():
         col.close(save=False)
 
 
-@pytest.mark.skip
 def test_push_is_nontrivial_when_pushed_changes_are_reverted_in_repository():
     """
     The following operation should be nontrivial:
@@ -1850,7 +1781,6 @@ def test_push_is_nontrivial_when_pushed_changes_are_reverted_in_repository():
         assert "ki push: up to date." not in out
 
 
-@pytest.mark.skip
 def test_push_changes_deck_for_moved_notes():
     MULTIDECK: SampleCollection = get_test_collection("multideck")
     runner = CliRunner()
@@ -1885,7 +1815,6 @@ def test_push_changes_deck_for_moved_notes():
         assert colnote.deck == "aa::dd"
 
 
-@pytest.mark.skip
 def test_push_is_trivial_for_committed_submodule_contents(tmp_path: Path):
     UNCOMMITTED_SM: SampleCollection = get_test_collection(
         "uncommitted_submodule_commits"
@@ -1926,7 +1855,6 @@ def test_push_is_trivial_for_committed_submodule_contents(tmp_path: Path):
         assert "ki push: up to date." in out
 
 
-@pytest.mark.skip
 def test_push_prints_informative_warning_on_push_when_subrepo_was_added_instead_of_submodule(
     tmp_path: Path,
 ):
@@ -1956,7 +1884,6 @@ def test_push_prints_informative_warning_on_push_when_subrepo_was_added_instead_
             assert "'git submodule add'" in out
 
 
-@pytest.mark.skip
 def test_push_handles_tags_containing_trailing_commas():
     COMMAS: SampleCollection = get_test_collection("commas")
     runner = CliRunner()
@@ -1980,7 +1907,6 @@ def test_push_handles_tags_containing_trailing_commas():
         push(runner)
 
 
-@pytest.mark.skip
 def test_push_correctly_encodes_quotes_in_html_tags(tmp_path: Path):
     BROKEN: SampleCollection = get_test_collection("broken_media_links")
     runner = CliRunner()

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -479,6 +479,7 @@ def test_clone_displays_nice_errors_for_missing_dependencies():
         assert os.path.isdir(HTML.repodir)
         F.rmtree(F.test(Path(HTML.repodir)))
         old_path = os.environ["PATH"]
+        logger.debug(f"{old_path = }")
 
         # In case where nothing is installed, we expect to fail on `tidy`
         # first.

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -497,11 +497,14 @@ def test_clone_displays_nice_errors_for_missing_dependencies():
         try:
             with pytest.raises(git.GitCommandNotFound) as raised:
                 tmp = F.mkdtemp()
-                assert "C:\\Program Files\\Git\\cmd;" in os.environ["PATH"]
-                os.symlink("/usr/bin/tidy", tmp / "tidy")
-                os.environ["PATH"] = str(tmp)
+
                 if sys.platform == "win32":
-                    os.environ["PATH"] = r"C:\ProgramData\chocolatey\bin\tidy.exe"
+                    git_path = "C:\\Program Files\\Git\\cmd;" in os.environ["PATH"]
+                    os.environ["PATH"] = os.environ["PATH"].replace(git_path, "")
+                else:
+                    os.symlink("/usr/bin/tidy", tmp / "tidy")
+                    os.environ["PATH"] = str(tmp)
+
                 clone(runner, HTML.col_file)
             error = raised.exconly()
         finally:

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -479,7 +479,6 @@ def test_clone_displays_nice_errors_for_missing_dependencies():
         assert os.path.isdir(HTML.repodir)
         F.rmtree(F.test(Path(HTML.repodir)))
         old_path = os.environ["PATH"]
-        logger.debug(f"{old_path = }")
 
         # In case where nothing is installed, we expect to fail on `tidy`
         # first.

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -497,6 +497,7 @@ def test_clone_displays_nice_errors_for_missing_dependencies():
         try:
             with pytest.raises(git.GitCommandNotFound) as raised:
                 tmp = F.mkdtemp()
+                assert "C:\\Program Files\\Git\\cmd;" in os.environ["PATH"]
                 os.symlink("/usr/bin/tidy", tmp / "tidy")
                 os.environ["PATH"] = str(tmp)
                 if sys.platform == "win32":

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -499,7 +499,7 @@ def test_clone_displays_nice_errors_for_missing_dependencies():
                 tmp = F.mkdtemp()
 
                 if sys.platform == "win32":
-                    git_path = "C:\\Program Files\\Git\\cmd;" in os.environ["PATH"]
+                    git_path = "C:\\Program Files\\Git\\cmd;"
                     os.environ["PATH"] = os.environ["PATH"].replace(git_path, "")
                 else:
                     os.symlink("/usr/bin/tidy", tmp / "tidy")

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -477,6 +477,8 @@ def test_clone_displays_nice_errors_for_missing_dependencies():
                 tmp = F.mkdtemp()
                 os.symlink("/usr/bin/tidy", tmp / "tidy")
                 os.environ["PATH"] = str(tmp)
+                if sys.platform == "win32":
+                    os.environ["PATH"] = r"C:\ProgramData\chocolatey\bin\tidy.exe"
                 clone(runner, HTML.col_file)
             error = raised.exconly()
         finally:

